src/dbghelp.rs

```diff
@@ -277,11 +294,8 @@ pub fn init() -> Result<Init, ()> {
         static LOCK: AtomicUsize = AtomicUsize::new(0);
         let mut lock = LOCK.load(SeqCst);
         if lock == 0 {
-            lock = CreateMutexA(
-                ptr::null_mut(),
-                0,
-                "Local\\RustBacktraceMutex\0".as_ptr() as _,
-            ) as usize;
+            let name = mutex_name();
+            lock = CreateMutexA(ptr::null_mut(), 0, name.as_ptr().cast::<i8>()) as usize;
```

Comment:
Does windows remove the mutex again once all open handles are closed or will this keep accumulating mutexes until the session ends?

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-createmutexa

> Use the [CloseHandle](https://learn.microsoft.com/en-us/windows/desktop/api/handleapi/nf-handleapi-closehandle) function to close the handle. The system closes the handle automatically when the process terminates. The mutex object is destroyed when its last handle has been closed.